bin/main.rb

```diff
@@ -0,0 +1,2 @@
+# !/usr/bin/env ruby
```

Comment:
- [ ] There should not be a space after the "#". It should be like this:

![image](https://user-images.githubusercontent.com/61479648/93762004-2b413e00-fbe5-11ea-83c4-0e9f943238f1.png)

- [ ] You need to run the "chmod 755" again. I know that it does not change the file, so it does not show up on Github, but this adds permissions to the file. Run this, and make a commit afterward. I did this and after it works. (instead of greeter.rb put your main.rb).

![image](https://user-images.githubusercontent.com/61479648/93761873-ef0ddd80-fbe4-11ea-8c40-78c81f50f7ad.png)